src/frontend/staff-portal/src/app/components/ticket-info/ticket-info.component.ts

```diff
@@ -635,11 +624,20 @@ export class TicketInfoComponent implements OnInit {
         this.initialDisputeValues.violationTicket.violationTicketCounts.forEach(violationTicketCount => {
 
```

Comment:
let countForm = this.form.get('violationTicket').get('violationTicketCount' + violationTicketCount.count.toString());
countForm.patchValue(violationTicketCount);
....
....